### PR TITLE
BevelButtWeld

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

File: weld.fs.js [hunk 1]
```diff
@@ -1016,67 +1016,81 @@ function sketchVButtWeld(context is Context, definition is map, thickness is Val
 function sketchBevelButtWeld(context is Context, definition is map, thickness is ValueWithUnits, profileSketch is Sketch)
 {
     var shape = definition.buttShape;
+    var offset = definition.buttOffset;
+    var angle = definition.buttAngle;
     var rootGap = definition.buttRootGap;
     var rootGapWidth = definition.buttRootGapWidth;
     var rootGapHeight = definition.buttRootGapHeight;
+            
+    if (side2)
```
[the remark this answers is not in the document]
Comment (emagdalenaC2i) [reply weld.fs.js]: Sorry it is in #43

[the remark this answers is not in the document]
Comment (mbartlett21) [reply weld.fs.js]: Do you want me to add this in?

[the remark this answers is not in the document]
Comment (emagdalenaC2i) [reply weld.fs.js]: No.  I did it in #43

[the remark this answers is not in the document]
Comment (mbartlett21) [reply weld.fs.js]: I have merged #43 and it doesn't seem to be there

[the remark this answers is not in the document]
Comment (emagdalenaC2i) [reply weld.fs.js]: It was in this commit https://github.com/mbartlett21/weld-featurescript/pull/43/commits/f34889637261d04dcc21822bea2180236ff1388c

[the remark this answers is not in the document]
Comment (mbartlett21) [reply weld.fs.js]: Ok, Github only seems to show changes from the time you branched this.

**Pull request merged**